### PR TITLE
Optionally pass an error handler for TreeView node selection

### PR DESCRIPTION
**Pull request by skateman**
The new `errorHandlers` binding allows us to set custom error handlers for various operations in the `TreeView`.

### CODE REVIEW

File: src/tree-view/treeViewComponent.ts [hunk 1]
```diff
@@ -102,11 +105,14 @@ export class TreeViewController {
       tail = [];
     }
 
+    console.log(this.errorHandlers);
```
Comment (karelhala) [inline src/tree-view/treeViewComponent.ts]: Remove `console.log` please.

Comment (skateman) [reply src/tree-view/treeViewComponent.ts]: OMG, sorry :D

**Comment by skateman** [comment]
@karelhala should be OK now!

**Comment by karelhala** [comment]
LGTM

**Pull request merged**